src/browser/web-preview.js

```diff
@@ -275,14 +275,17 @@ define((require, exports, module) => {
     ]);
 
   const viewInEditMode = (loaders, pages, selected, theme, address) =>
-    viewContainer(theme, ...viewPreviews(loaders, pages, selected, address));
+    viewContainer(theme, ghostPreview, ...viewPreviews(loaders, pages, selected, address));
```

Comment:
Always showing ghost preview to avoid issues with "jumping" caused by rendering edit mode when transitioning from `create-web-view -> show-web-view`.